src/core/core.scale.js

```diff
@@ -537,6 +541,39 @@ module.exports = Element.extend({
 		return rawValue;
 	},
 
+	_parseValue: function(raw) {
+		var value = +this.getRightValue(raw);
+		var start, end, min, max;
+
+		if (helpers.isArray(value)) {
+			start = value[0];
+			end = value[1];
+			min = Math.min(start, end);
+			max = Math.max(start, end);
+		} else {
+			start = undefined;
+			end = value;
+			min = value;
+			max = value;
+		}
+
+		return {
+			min: min,
+			max: max,
+			start: start,
+			end: end
+		};
+	},
+
+	getScaleLabel: function(rawValue) {
+		var v = this._parseValue(rawValue);
+		if (v.start !== undefined) {
+			return v.min + ' ; ' + v.max;
```

Comment:
Can we make this `return v.start + ' - ' + v.end;`

Reply:
yes we can if we dont want to force standartized label for biderectional syntaxis of float-bars - 2,4 and 4,2

Reply:
Not a good idea ([see this comment](/chartjs/Chart.js/pull/5262#discussion_r186955297)), I would absolutely avoid `v.start + ' - ' + v.end;`.

Reply:
Actually, there is two points in @benmccann comment: I agree with replacing `v.min` and `v.max` by `start` and `end`, but I disagree with replacing ` ; ` by ` - `. I'm not against `return '[' + v.start + ', ' + v.end + ']';` as suggested in the other PR (or `(start, end)` instead of `[start, end]` as currently implemented for [the bubble chart](/chartjs/Chart.js/blob/master/src/controllers/controller.bubble.js#L38)) but we should not use a word as delimiter because we will need to introduce i18n.

Reply:
'[' + v.start + ', ' + v.end + ']'; would be nice because it's standard interval notation and it's what the user passed in

Reply:
I also like `[start, end]` better.